api/libs/utils/src/geo/bbox.ts

```diff
@@ -0,0 +1,38 @@
+import { BBox } from 'geojson';
+/**
+ * Utility functions related to lower-level interaction with bbox operations.
+ *
+ * @debt This should be moved to a self-standing
+ * @debt there is also a mismatch in terms of the bbox creation [xmax,xmin,ymax,ymin] vs [xmin,xmax,ymin,ymax]
+ */
+
+/**
+ * conversion operation between bbox [xmin, ymin, xmax, ymax]
+ * to Nominatim bbox  [xmin, xmax, ymin, ymax].
+ *
+ */
+export function bbox2Nominatim(bbox: BBox): BBox {
```

Comment:
you could consider aliasing `BBox` as `NominatimBBox` to improve type expressiveness in function signatures, even though in practice it's still just an array of 4 or 6 numbers.

Reply:
nothing to do here though, unless you wish to, of course 😃